src/cityreader/cityreader.js

```diff
@@ -0,0 +1,29 @@
+const csv = require("csvtojson");
+class City {
+  constructor(name, lat, lon) {
+    this.name = name;
+    this.lat = lat;
+    this.lon = lon;
+  }
+  speak() {
+    return `${this.name}, ${this.lat}, ${this.lon}`;
+  }
+}
+
+const cities = [];
+
```

Comment:
Great job converting the printed output results to the same format expected from the test utilizing Javascript.